src/index.d.ts

```diff
@@ -17,7 +17,7 @@ declare namespace preact {
 		/**
 		 * ref is not guaranteed by React.ReactElement, for compatiblity reasons
 		 * with popular react libs we define it as optional too
-		*/
+		 */
```

Comment:
You might have some global prettier config

Reply:
I'm not sure what is causing this, I would have thought that no other prettier config ran from my workspace after the pre-commit's `lint-staged`.

That said, it seems to be consistent with the other comment blocks.

Reply:
All right, no worries ;D looks better either way